src/main/java/org/sunbird/portal/department/model/Rollup.java

```diff
@@ -0,0 +1,41 @@
+package org.sunbird.portal.department.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "l1" })
+
+public class Rollup {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.